app/routes.js

```diff
@@ -1,13 +1,18 @@
 // @flow
 import React from 'react';
-import { Route } from 'react-router';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Switch, Route } from 'react-router';
 import App from './containers/App';
 import HomePage from './containers/HomePage';
 import CounterPage from './containers/CounterPage';
 
-export default (
-  <Route path="/" component={App}>
-    <Route path="/" component={HomePage} />
-    <Route path="/counter" component={CounterPage} />
-  </Route>
+export default () => (
+  <Router>
+    <App>
+      <Switch>
+        <Route path="/counter" component={CounterPage} />
+        <Route path="/" component={HomePage} />
```

Comment:
I'm not happy how the path are defined. I expected to be able to use `<Route exact path="/" component={Homepage}>` but location.pathname is '/Users/path/electron-react-boilerplate/app/app.html'. This would work but feels wrong: `<Route path="*/app.html" component={HomePage} />`